tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md

```diff
@@ -65,7 +65,7 @@ Cloud Platform (GCP).
     [HyperSQL database][hyper]. You will now switch from HyperSQL to using Cloud
     SQL as your database.
 
-1. Note: [spring-petclinic](spring-petclinic) provides a copy of the source code
+1. Note: [spring-petclinic][source_path] provides a copy of the source code
```

Comment:
Sorry to do this, but for consistency with other tutorials, can you please use dashes rather than underscores in your bookmark names?